### ISSUE TITLE
Transparent text in console when enrolled for flags testing

### ISSUE THREAD

**Issue by mabhub** [gatsbyjs/gatsby#29567]
## Description

When a site is (randomly?) picked up for feature testing, this message is displayed:

![image](https://user-images.githubusercontent.com/1846633/108377730-9c246a80-7204-11eb-8f8e-a72ef77eb9d8.png)

Any further console text is present **but** *transparent*:

![image](https://user-images.githubusercontent.com/1846633/108377850-b9593900-7204-11eb-868b-6a6ad01a6954.png)

If I set   `flags: { DEV_SSR: false },` in `gatsby-config.js`, the same behavior still occurs:

![image](https://user-images.githubusercontent.com/1846633/108378377-3ab0cb80-7205-11eb-9abd-714f27bb1cc3.png) 
![image](https://user-images.githubusercontent.com/1846633/108378436-48665100-7205-11eb-863e-f987eeea16da.png)
*(text is selected only to emphasize existing transparent content)*

#### But!

Whenever the terminal window is wide enough for the *flags explanation lines* to not be croped, the remaining text keep being in proper color:

![image](https://user-images.githubusercontent.com/1846633/108379593-5ff20980-7206-11eb-9004-49f4320b6f74.png)

### Steps to reproduce

I tryd reproducing *feature testing conditions* without success…

### Expected result

Readable console with any terminal width

### Actual result

Unreadable console.

### Environment

<details>
<summary>
`gatsby info --clipboard`
</summary>

```
  System:
    OS: Linux 5.8 Ubuntu 20.04.2 LTS (Focal Fossa)
    CPU: (12) x64 Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz
    Shell: 5.8 - /usr/bin/zsh
  Binaries:
    Node: 14.15.4 - ~/.nvm/versions/node/v14.15.4/bin/node
    npm: 7.5.4 - ~/.nvm/versions/node/v14.15.4/bin/npm
  Browsers:
    Chrome: 88.0.4324.150
    Firefox: 86.0b9
  npmPackages:
    gatsby: ^2.32.3 => 2.32.3 
    gatsby-source-filesystem: ^2.11.0 => 2.11.0 
    gatsby-theme-material-ui: ^1.0.13 => 1.0.13 
    gatsby-transformer-remark: ^2.16.0 => 2.16.0 
  npmGlobalPackages:
    gatsby: 2.31.1
```

*(The same issue persits when using (vanilla) `bash` instead of `zsh`)*

</details>

## Possible cause

Maybe a *special color code* for text coloring is not properly reset when ending/closing/corresponding special code is not rendered in terminal (when cropped by terminal window).

*(I did not find any existing similar issue but maybe I didn't searched enough…)*

**Comment by LekoArts** [comment]
This sounds like a duplicate of https://github.com/gatsbyjs/gatsby/issues/29293 and this was fixed. So can you make sure you're on the latest version?

**Comment by mabhub** [comment]
It seems that your totally right about duplication ! *(maybe did I search only in opened issues…  )*

But also, I think I'm already using the latest release *(I did `gatsby new` with `hello-world` starter, then `npm i gatsby@latest` today)*…  Maybe has the fix not already been shipped in a release.

But it may happens pretty soon.
In the meantime, I'll be disabling colors as suggested!

**Comment by ascorbic** [comment]
This is published in `gatsby@2.32.4`
